### DISCUSSION TITLE
Customer Service and Support

### DISCUSSION THREAD

**Discussion by sympmarc** [MicrosoftDocs/microsoft-365-community#383] [Ideas]
How can you ask for support well? 

How can you provide it well to people in *your* organization?

Hint: there are common characteristics.

**Comment by tomresing** [comment]
For Microsoft 365, the best way to start a support request is from the Microsoft 365 admin center. Start by selecting the “?” to open a help dialog. Then, describe your issue. There may be known answers that could be related to specific situations for your organization. If not, you’re starting a support request. Requests are tracked by Microsoft and can be passed along from one support resource to another. Microsoft looks at support requests both individually and in groups by similar conditions. The solution to an issue you’ve come across might be the result of a number of requests that together identify a pattern. Does this issue impact your work? Include details about the effect and severity of the impact. Support issues can range in impact. Is it stopping a large group of people from getting things done? Include the numbers and timelines. Issues that don’t stop work are still important, they may just have a different level of urgency in response time. Request an escalation when the issue is urgent and not improving.

**Comment by sympmarc** [comment]
"Then, describe your issue."

That's where a lot of support requests go south, IMO. Reporting an issue well is a skill. For example, here in this repo, we've got several issue templates which ought to serve well for the type of issue you're reporting or suggesting.
![image](https://user-images.githubusercontent.com/1295627/101388633-3d4e8680-388e-11eb-9e2c-aeff034868f8.png)

Those templates ask for different information based on the type of issue. It's surprising how few people actually use the template when they contribute an issue.

More broadly, many customer service sites ask pretty specific questions. Make sure you answer those questions clearly and well. When you describe your issue, remember that the person on the other end has probably no context about your situation or what you've done leading up to the issue. Reporting an issue well means you'll get a much better initial reply.

**Comment by tomresing** [comment]
@sympmarc I think it would be great if we had contributors with recent experience. I hadn't sent in a support request in many years, so I made one this morning.
![image](https://user-images.githubusercontent.com/2660081/101544314-5290e700-395a-11eb-8bb1-b163706d9798.png)

**Reply by sympmarc** [reply]
I'll be interested to hear how it goes. :)

**Reply by sympmarc** [reply]
How did that support request go, @tomresing?

**Comment by tomresing** [comment]
Partial success. They let me know it was related to a bug that has a fix. However, that was still frustrating to find out. And, I’m still waiting for the fix to land in my organization.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by sympmarc** [reply]
Bummer. "My organization", eh? :)

**Comment by tomresing** [comment]
A non-profit I volunteer for.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
